tests/copp/test_copp.py

```diff
@@ -395,19 +414,19 @@ def _setup_multi_asic_proxy(dut, creds, test_params, tbinfo):
         return
 
     logging.info("Adding iptables rules and enabling eth0 port forwarding")
-    http_proxy, https_proxy = copp_utils._get_http_and_https_proxy_ip(creds)
     # Add IP Table rule for http and ptf nn_agent traffic.
     dut.command("sudo sysctl net.ipv4.conf.eth0.forwarding=1")
-    mgmt_ip = dut.host.options["inventory_manager"].get_host(dut.hostname).vars["ansible_host"]
-    # Add Rule to communicate to http/s proxy from namespace
-    dut.command("sudo iptables -t nat -A POSTROUTING -p tcp --dport 8080 -j SNAT --to-source {}".format(mgmt_ip))
-    dut.command("sudo ip -n {} rule add from all to {} pref 1 lookup default".format(test_params.nn_target_namespace, http_proxy))
-    if http_proxy != https_proxy:
-        dut.command("sudo ip -n {} rule add from all to {} pref 2 lookup default".format(test_params.nn_target_namespace, https_proxy))
+
+    if not test_params.swap_syncd:
+        http_proxy, https_proxy = copp_utils._get_http_and_https_proxy_ip(creds)
+        mgmt_ip = dut.host.options["inventory_manager"].get_host(dut.hostname).vars["ansible_host"]
+        # Add Rule to communicate to http/s proxy from namespace
+        dut.command("sudo iptables -t nat -A POSTROUTING -p tcp --dport 8080 -j SNAT --to-source {}".format(mgmt_ip))
+        if http_proxy != https_proxy:
+            dut.command("sudo ip -n {} rule add from all to {} pref 3 lookup default".format(test_params.nn_target_namespace, https_proxy))
```

Comment:
Why do we not add the perf 1 and perf 2 rules that was there before ?

Reply:
good catch @judyjoseph . We don;t need all this change now.